lib/lambda_ethereum_consensus/fork_choice/store.ex

```diff
@@ -36,14 +38,14 @@ defmodule LambdaEthereumConsensus.ForkChoice.Store do
 
   @spec has_block?(SszTypes.root()) :: boolean()
   def has_block?(block_root) do
-    [blocks] = get_store_attrs([:blocks])
-    Map.has_key?(blocks, block_root)
+    block = get_block(block_root)
+    block != nil
```

Comment:
This could be optimized even further by making `get_block` return a boolean instead of the whole block.

Reply:
We could further optimize it, but im keeping it simple for now (we also want to be able to get a full block)